plugins/button/plugin.js

```diff
@@ -51,7 +52,10 @@
 	( CKEDITOR.env.hc ? '&#9660;' : '' ) +
 		'</span>';
 
+	var templateToggle = ' aria-pressed="false"';
+
 	var btnArrowTpl = CKEDITOR.addTemplate( 'buttonArrow', templateArrow ),
+		toggleBtnTpl = CKEDITOR.addTemplate( 'toggleButton', templateToggle ),
```

Comment:
I'm not sure the toggle template is a good candidate for moving it into the templates store. Do you see any use case where it would make sense to customize this template using the "template" event?

Reply:
I've not even know we have such an event :see\_no\_evil: I simply used a template here as all other button's parts were templates. But you're right, there's no need to have a template for the single attribute.